gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/base.py.j2

```diff
@@ -33,6 +33,12 @@ try:
         gapic_version=pkg_resources.get_distribution(
             '{{ api.naming.warehouse_package_name }}',
         ).version,
+        {% if 'grpc' not in opts.transport %}
+        grpc_version=None,
+        {% endif %}
+        {% if 'rest' in opts.transport %}
+        rest_version=pkg_resources.get_distribution("requests").version,
```

Comment:
Can we use `requests.__version__` instead? We're trying to move away from using `pkg_resources` as it may not work depending on how the library is installed.

```py
Python 3.8.3 (default, Jun 15 2020, 16:29:21) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import requests
>>> requests.__version__
'2.24.0'
```

Reply:
Sure, if it is the better way of doing it!

Reply:
Actually that implies that I would have to import `requests` module explicitly in `base.py` files. If that is ok with you I'll do it, but please confirm.

Reply:
`requests` is a requirement of `google-api-core` so I am OK with importing it explicitly in the GAPIC surface.

https://github.com/googleapis/python-api-core/blob/c4ff747db17cdeedb0eb2741b308c7b2d2a596fb/setup.py#L35